redisenterprise/assets/configuration/spec.yaml

```
@@ -0,0 +1,33 @@
+name: RedisEnterprise
+files:
+- name: redisenterprise.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/http 
+    - template: init_config/default
+  - template: instances
+    options:
+    - name: host
+      required: true
+      description: The RedisEnterprise host
+      value:
+        type: string
+        example: myrediscluster.example.com
+    - name: port
+      required: false
+      description: The RedisEnterprise API port - default 9443
+      value:
+        type: integer
+        example: 9443
+    - name: event_limit
+      required: false
+      description: Number of events to fetch - default 50
+      value:
+        type: integer
+        example: 50
+    - template: instances/http
+       overrides:
+         username.description: The RedisEnterprise API user
+         username.value.example: redisadmin@example.com
```

```suggestion
    - template: instances/http
      overrides:
          username.description: The RedisEnterprise API user
          username.value.example: redisadmin@example.com
```
Sorry about this, seems my prior suggestion was missaligned. It seems github suggest editor does not show spaces right.
If you open this in a monospaced editor you should see that `overrides` is aligned with `template` and the inner elements of overrides should be indented four spaces with respect of the header.

After applying this change you will need to run `ddev -x validate config -s redisenterprise` again

---

@hithwen thanks - that was confusing with the spacing :)  All fixed and config validate run again